packages/opentelemetry-instrumentation-fetch/README.md

```diff
@@ -17,23 +17,22 @@ npm install --save @opentelemetry/instrumentation-fetch
 
 ```js
 'use strict';
-import { ConsoleSpanExporter, SimpleSpanProcessor } from '@opentelemetry/tracing';
-import { WebTracerProvider } from '@opentelemetry/web';
-import { FetchInstrumentation } from '@opentelemetry/instrumentation-fetch';
-import { ZoneContextManager } from '@opentelemetry/context-zone';
```

Comment:
I think you should keep the import of instrumentations, processor/exporter and context-zone